Resources/Prototypes/Entities/Mobs/NPCs/funnyayadoll.yml

```diff
@@ -0,0 +1,89 @@
+- type: entity
+  name: poltergeist
+  id: Mobayadoll
+  parent: SimpleMobBase
+  description: AWW its so cute.
+  components:
+   - type: Vocal
+    maleScream:
+      collection: doll
+    femaleScream:
+      collection: doll
+  - type: PsionicBonusChance
+      multiplier: 10
+      warn: false
+ - type: Carriable
+ - type: Loadout
+    prototypes: [PassengerGear]
+  - type: CombatMode
+   canDisarm: true
+  - type: MovementSpeedModifier
+    baseWalkSpeed : 4.00
+    baseSprintSpeed : 7.00
+  - type: InputMover
+  - type: MobMover
+  - type: Reactive
+    groups:
+      Flammable: [Touch]
+      Extinguish: [Touch]
+  - type: Faction
+    factions:
+    - SimpleHostile
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/ayadoll.rsi/dollspritesfull.png
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: poltergeisst
```

Comment:
This has poltergeist with two `s`, and I don't see a `poltergeisst` state outlined in the `meta.json` for `ayadoll.rsi`. Additionally there is no `poltergeist.png` in the RSI, so it is invalid. It would have to be `dollspriteclean` given the current directory layout.